maestrowf/datastructures/core/executiongraph.py

```diff
@@ -602,16 +602,21 @@ def _execute_record(self, record, adapter, restart=False):
     def write_status(self, path):
         """Write the status of the DAG to a CSV file."""
         header = "Step Name,Workspace,State,Run Time,Elapsed Time,Start Time" \
-                 ",Submit Time,End Time,Number Restarts"
+                 ",Submit Time,End Time,Number Restarts, Job ID"
```

Comment:
I feel as though the `JobID` should be after the `Step Name` since they're both identifiers. Thoughts?

Reply:
I think that's more appropriate too